addons/hr_timesheet/models/project_update.py

```diff
@@ -34,6 +34,6 @@ def create(self, vals_list):
             update.write({
                 "uom_id": encode_uom,
                 "allocated_time": round(project.allocated_hours / ratio),
-                "timesheet_time": round(project.total_timesheet_time / ratio),
+                "timesheet_time": round(project.sudo().total_timesheet_time / ratio),
```

Comment:
why is it needed? 🤔 

I surprise to see it is needed here and not for the line above for instance.

Reply:
okidoki, understood, understood. 🤖